File: StarWars_Java/src/main/java/Personnage.java
package starwars;

public class Personnage{
	private String Nom;
	private String Prenom;
	
	public Personnage(){	
	}
	
	public Personnage(String Nom, String Prenom) {
        this.Nom = Nom;
        this.Prenom = Prenom;
    }

    public String toString() {
        return "Classe : " + getClass().getSimpleName() +
        		"\nNom : " + Nom +
        		"\nPrenom : " + Prenom ;
    }

	
}

File: StarWars_Python/Personnage.py
class Personnage :

    def __init__(self, first_name=None, last_name=None):
        self.first_name = first_name
        self.last_name = last_name

    def __str__(self):
        return f"Personnage [Prénom: {self.first_name}, Nom: {self.last_name}]"
